console/src/App/AppContextProvider.js

```diff
@@ -186,7 +186,8 @@ class AppContextProvider extends Component {
 	};
 
 	loadUserSettings = (userId) => {
-		return JSON.parse((global.localStorage && global.localStorage.getItem(`user${userId}`)) || '{}') || this.getDefaultUserSettings(userId);
+		const user = JSON.parse((global.localStorage && global.localStorage.getItem(`user${userId}`)) || '{}');
+		return Object.keys(user).length !== 0 ? user : this.getDefaultUserSettings(userId);
```

Comment:
can you give me a test scenario in order to catch the error that was solved? i couldn't find it myself